backend/infrastructures/transform/transform.py

```diff
@@ -0,0 +1,8 @@
+import modules.transform.transform as transform
+import pandas as pd
+
+class TransformationHandler():
+    def process(self, df: pd.DataFrame):
+        if df == None:
```

Comment:
What happened here? This should never never be none.

Reply:
agreed. I removed it.